autoconfigure/README.md

```diff
@@ -0,0 +1,36 @@
+# Configuration
+
+## Storage configurations
+
+| Configuration | Description | Default |
+|---------------|-------------|---------|
+| `KAFKA_STORE_SPAN_CONSUMER_ENABLED` | Enable partitioning of incoming span batches into individual spans keyed by trace ID. | `true` |
+| `KAFKA_STORE_AGGREGATION_ENABLED` | Enable aggregation spans into traces and then into dependency links. Currently this has to be initiated with span consumer or storage, or initiate manually via `curl /health` | `true` |
+| `KAFKA_STORE_SPAN_STORE_ENABLED` | Enable storage for spans, traces, dependencies, tags, and service names. | `true` |
+| `KAFKA_STORE_BOOTSTRAP_SERVERS` | Kafka bootstrap servers | `localhost:9092` |
```

Comment:
We should think about if re-using the other kafka bootstrap servers variable makes sense.. and any difference between the default zipkin topic and the spans one is notable enough to mention somewhere (if not already mentioned).

ideally we have only a few KAFKA_STORAGE_X variables and centralize kafka producer/consumer related properties under the KAFKA_X namespace.

Reply:
> We should think about if re-using the other kafka bootstrap servers variable makes sense..

I thought about this at the beginning but if we do that, then it will force to always consume spans collected, when we might want to define that on a specific node, and do the aggregation/storage later.

> any difference between the default zipkin topic and the spans one is notable enough to mention somewhere (if not already mentioned).

I'll add the specifics on the docs. Main difference between them is partitioning (trace ID as key). Have been thinking about it and I believe is better to have this separate from Kafka Sender as it could became much slower, as we already do batching there and this will involve and additional loop to prepare records and batch them with Kafka; unless I'm missing something.

Reply:
last config list is reduced to `KAFKA_X` focused on connection and topic names, and `KAFKA_STORAGE_X` on few timeout/window configs :)